docs/getting-started/getting-started-azure-cosmos-db.md

```diff
@@ -148,8 +143,6 @@ Once you have added the `Book` entity, the `entities` object of the configuratio
  "entities": {
     "Book": {
       "source": "books",
-      "rest": false,
-      "graphql": true,
```

Comment:
why are we removing this? graphql is required otherwise it won't work.

Reply:
verified it by running without those fields and it worked correctly. After looking into the code i found out that we are only
Skipping the creation of the GraphQL object for an entity when `graphql=false` is explicitly mentioned in the config.